lld/ELF/Thunks.cpp

```diff
@@ -1519,6 +1535,39 @@ bool PPC64LongBranchThunk::isCompatibleWith(const InputSection &isec,
   return rel.type == R_PPC64_REL24 || rel.type == R_PPC64_REL14;
 }
 
+// Hexagon Target Thunks
+static uint64_t getHexagonThunkDestVA(Ctx &ctx, const Symbol &s, int64_t a) {
+  uint64_t v = s.isInPlt(ctx) ? s.getPltVA(ctx) : s.getVA(ctx, a);
+  return SignExtend64<32>(v); // FIXME: sign extend to 64-bit?
```

Comment:
Why is there a FIXME?

Reply:
It was...sorry!  Removed for real now in 44bfa0463657b0abf34190eb491cdbf41b3bf50d